### ISSUE TITLE
Feature Request: Ungoogled Chromium

### ISSUE THREAD

**Issue by ylluminate** [dmarmor/epichrome#198]
Now that we're going down the Chromium rabbit hole a bit, it seems to me that it would be beneficial to offer a more static and "free" version of Chromium as another option:
https://chromium.woolyss.com/
![ungoogled](https://p153.p0.n0.cdn.getcloudapp.com/items/p9u5yG9O/Image+2020-01-27+at+1.59.54+AM.png)

This version seems to be fairly robust, but would probably need to have this plugin also incorporated into distribution:
https://github.com/NeverDecaf/chromium-web-store

**Comment by dmarmor** [comment]
The best way I'll be able to support this (or any other variant) is that I have plans to basically allow users like you who want customization over your engine a third engine option of "Custom." If you chose this when creating (or updating) an app, it would bring up a file chooser and you'd then select the path to your engine app, which could be anything that's basically Chromium-compatible. Making sure that was true would be up to the user, so it would be more of an advanced or experimental feature.

As far as what's bundled with Epichrome, I had to choose what seemed like the best combination of seamlessness for most users and robustness, which is how I ended up with the flavor I did.
